test/unit_test/automl_test.cc

```diff
@@ -88,6 +88,7 @@ using namespace ut_helper;
 // config 0 is hard-coded to be empty interactions for now.
 bool weights_offset_test(vw& all, multi_ex&)
 {
+  VW::test_red::tr_data* tr = ut_helper::get_automl_data(all);
```

Comment:
this is not used?

Reply:
I'll take that out, I was using it to see which interactions were live